devices/Mcp7940xx/tests/TestHelper.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Iot.Device.Common;
```

Comment:
and please also add the standard header here as well

Reply:
Done.